app/controllers/users_controller.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  include UserSettable
+
+  before_action lambda {
```

Comment:
Why do we need it?

Reply:
@kth-wtag Apu, I used this module to separate the functionality to find and set `user` for all the controllers where needed. It was not really necessary because finding the `user` is almost trivial in most of the cases. I just did it in this way to have some practice of `separation of concerns`.

Reply:
> @kth-wtag Apu, I used this module to separate the functionality to find and set `user` for all the controllers where needed. It was not really necessary because finding the `user` is almost trivial in most of the cases. I just did it in this way to have some practice of `separation of concerns`.

I thunk you can remove this functionality and do it more rails way.